src/LibraryManager.Vsix/UI/Controls/Library.xaml.cs

```diff
@@ -14,22 +14,50 @@ namespace Microsoft.Web.LibraryManager.Vsix.UI.Controls
     public partial class Library : INotifyPropertyChanged
     {
         public static readonly DependencyProperty CaretIndexProperty = DependencyProperty.Register(
-            nameof(CaretIndex), typeof(int), typeof(Library), new PropertyMetadata(default(int), SearchCriteriaChanged));
+            nameof(CaretIndex), typeof(int), typeof(Library), new PropertyMetadata(default(int)));
 
         public static readonly DependencyProperty SearchServiceProperty = DependencyProperty.Register(
-            nameof(SearchService), typeof(Func<string, int, Task<CompletionSet>>), typeof(Library), new PropertyMetadata(default(Func<string, int, Task<CompletionSet>>), SearchCriteriaChanged));
+            nameof(SearchService), typeof(Func<string, int, Task<CompletionSet>>), typeof(Library), new PropertyMetadata(default(Func<string, int, Task<CompletionSet>>)));
 
         public static readonly DependencyProperty SelectedItemProperty = DependencyProperty.Register(
             nameof(SelectedItem), typeof(Completion), typeof(Library), new PropertyMetadata(default(Completion)));
 
         public static readonly DependencyProperty TextProperty = DependencyProperty.Register(
-            nameof(Text), typeof(string), typeof(Library), new PropertyMetadata(default(string), SearchCriteriaChanged));
+            nameof(Text), typeof(string), typeof(Library), new PropertyMetadata(default(string)));
 
         private int _version;
 
         public Library()
         {
             InitializeComponent();
+
+            this.Loaded += new RoutedEventHandler(LibrarySearchBox_Loaded);
```

Comment:
>RoutedEventHandler [](start = 31, length = 18)

I'm assuming this doesn't work as just:

this.Loaded += LibrarySearchBox_Loaded

?